### PR TITLE
added trust type

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9242
According to "Configure Active Directory" section of the article, it applies to Certificate trust deployments

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

**Review by illfated — APPROVED** [review]
Looks good to me.

File: windows/security/identity-protection/hello-for-business/hello-feature-dual-enrollment.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ ms.reviewer:
 * Hybrid and On-premises Windows Hello for Business deployments
 * Enterprise Joined or Hybrid Azure joined devices
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-feature-dual-enrollment.md:25]:
```suggestion
* Enterprise joined or Hybrid Azure joined devices
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor additional edit, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by illfated** [comment]
#sign-off

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mapalko** [comment]
I don't see a mention in the article that this requires cert trust. I am checking with some engineers on this one. Please do not merge.

**Comment by MaratMussabekov** [comment]
Dear @mapalko ,
The "Configure Active Directory to support Domain Administrator enrollment" section of the article describes the actions that required for certificate trust only.
Thank you

**Pull request merged**